DOCS-10131-[ION-8744] Double static IP for Cloudhub DLBs (#456)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cloudhub-dedicated-load-balancer.adoc

You can assign up to four load balancer units to a DLB.
A DLB with four load balancer units assigned has eight workers.

== Static IP Addresses

If you enable static IP addresses for a DLB, the number of static IP addresses allocated to the DLB is equal to double the number of workers.
For example, if your DLB has four workers, the DLB has eight associated static IP addresses.

Half the static IPs (equal to the number of worker instances) are in use at any given time.
When you restart or update the DLB, the DLB switches to use the group of unused static IP addresses and transitions the first group of static IP addresses to the unused state.

Switching between the two groups of static IP addresses enables zero-downtime restarts and updates, ensuring that no transactions are dropped or lost as long as the transaction times are shorter than 600s or the value of *Mule Application Response Timeout* (whichever is smaller).

> Since, static IPs will change when restarting or updating your DLB, it is recommended to use DLB's DNS address rather than the IP addresses when calling it from an external service.

Author: @hannanelson @evinaku Could you please review this line once? 
We are trying to tell customers to not use IP addresses directly when accessing resources behind a DLB and instead use the DLBs DNS name/address that is available through the API/UI

> Since, static IPs will change when restarting or updating your DLB, it is recommended to use DLB's DNS address rather than the IP addresses when calling it from an external service.

Reviewer: ```suggestion
Because static IP addresses change when the DLB restarts, to call the DLB from an external service, use the DNS address of the DLB rather than the IP address.
```

Reviewer: (you have to restart to update the DLB, right?)

Author: Restart is part of the update operation

You can assign up to four load balancer units to a DLB.
A DLB with four load balancer units assigned has eight workers.

== Static IP Addresses

If you enable static IP addresses for a DLB, the number of static IP addresses allocated to the DLB is equal to double the number of workers.
For example, if your DLB has four workers, the DLB has eight associated static IP addresses.

Half the static IPs (equal to the number of worker instances) are in use at any given time.
When you restart or update the DLB, the DLB switches to use the group of unused static IP addresses and transitions the first group of static IP addresses to the unused state.

> Switching between the two groups of static IP addresses enables zero-downtime restarts and updates and ensures no transactions are dropped or lost as long as the transactions are shorter than 600s or Mule Application Response Timeout (whichever is smaller)

Reviewer: ```suggestion
Switching between the two groups of static IP addresses enables zero-downtime restarts and updates, ensuring that no transactions are dropped or lost as long as the transaction times are shorter than 600s or the value of *Mule Application Response Timeout* (whichever is smaller).
```

All newly created DLBs with static IP use this static IP model. 

> For existing DLBs that use static IP addresses, the original static IP addresses will continue to be used on restart of the DLB unless opted-in using the *Update* button. This change cannot be reverted and will be enforced in the future. The Runtime Manager *Load Balancers* page displays *Update Required* until you update the DLB:

Reviewer: ```suggestion
Existing DLBs that use static IPs continue to use the original static IP addresses after restarting the DLB, unless you click the *Update* button.
After clicking *Update*, the DLB uses the new static IP model and you cannot revert to the old model.
The Runtime Manager *Load Balancers* page displays *Update Required* until you update the DLB:
```

You can assign up to four load balancer units to a DLB.
A DLB with four load balancer units assigned has eight workers.

== Static IP Addresses

If you enable static IP addresses for a DLB, the number of static IP addresses allocated to the DLB is equal to double the number of workers.
For example, if your DLB has four workers, the DLB has eight associated static IP addresses.

Half the static IPs (equal to the number of worker instances) are in use at any given time.
When you restart or update the DLB, the DLB switches to use the group of unused static IP addresses and transitions the first group of static IP addresses to the unused state.

> Switching between the two groups of static IP addresses enables zero-downtime restarts and updates, ensuring that no transactions are dropped or lost as long as the transaction times are shorter than 600s or the value of *Mule Application Response Timeout* (whichever is smaller).

Reviewer 2: Does 600s = 600 seconds? I'd spell it out.

=== Considerations

> * Static IPs will change when restarting or updating your DLB, it is recommended to use DLB's DNS address rather than the IP addresses when calling it from an external service. Ensure that the client includes the necessary retry mechanisms to account for DNS switch.

Reviewer 2: ```suggestion
* Because static IPs change when you restart or update your DLB, use the DLB's DNS address rather than the IP addresses when calling the DLB from an external service. Ensure that the client includes the necessary retry mechanisms to account for the DNS switch.
```

> * To avoid connection issues when the load balancers is restarting, it's recommended to honor the DNS TTL (30 seconds), including when using connection keep alive. The connections will be closed after the new workers are online and the TTl expires.

Reviewer 2: ```suggestion
* To avoid connection issues when the load balancer is restarting, adhere to the DNS TTL (30 seconds), including connection keep alive. The connections are closed after the new workers are online and the TTL expires.
```

[NOTE]

> For DLBs created before October 2021, click *Update* to enable the feature. After clicking *Update*, the DLB uses the static IP model described above, and you cannot revert to the old model.

Reviewer 2: ```suggestion
For DLBs created before October 2021, click *Update* to enable the feature. After clicking *Update*, the DLB uses the static IP model described earlier, and you cannot revert to the former model.
```

> For information on updating DLBs, see

Reviewer 2: ```suggestion
For information about updating DLBs, see
```
